src/i18n/locales/en.json

```diff
@@ -488,94 +488,102 @@
         "viewQueue": "view queue"
     },
     "setting": {
-        "accentColor": "accent color",
         "accentColor_description": "sets the accent color for the application",
```

Comment:
Why were all these keys changed? Some auto-sort?

Reply:
Yeah - I sorted the keys as they appeared to be trying to follow an alphabetical format - no keys were removed, just moved around :)